inference/run_api.py

```diff
@@ -165,6 +205,16 @@ def gpt_tokenize(string: str, encoding) -> int:
     return num_tokens
 
 
+def get_model_id(model_name: str) -> str:
+    """Returns normalized model id from abitrary serving model name.
+    E.g., abitrary_path/deepseek-coder-33b -> deepseek.
+    """
+    if "deepseek" in model_name:
+        return "deepseek"
+    if "codellama" in model_name:
+        return "codelllma"
+
```

Comment:
What happens if none of the conditions are triggered?

Reply:
Have resolved it.